articles/multifactor-authentication/index2.md

```diff
@@ -15,49 +15,45 @@ Multifactor Authentication (MFA) is a method of verifying a user's identity by r
+Auth0 supports sending an SMS with a one-time password code to be used for another step of verification.
 
-* Google Authenticator;
-* Duo Security.
+[Click here to learn more about enabling SMS](/multifactor-authentication/guardian/admin-guide#support-for-sms)
 
-You may enable these options via the Auth0 Management Dashboard. To navigate to the appropriate configuration page, navigate to the Multifactor Auth page. You will see the following message, along with the link to the appropriate page:
+## MFA Using Google Authenticator
 
-> Auth0 also supports plugging in other multifactor providers. If you already have your own provider, click here to use a different provider.
+Google Authenticator is a mobile app that generates 2-step verification codes. This creates a one-time use password that is used as the second factor after your user has attempted to log in with their Google credentials.
 
-#### MFA Using Google Authenticator
+[Click here to learn more about enabling Google Authenticator](/multifactor-authentication/google-auth)
 
-Google Authenticator allows you to request six- to eight-digit one-time use password as the second factor after your user has attempted to log in with their Google credentials.
-
-You will find further instructions on enabling this feature [here](/multifactor-authentication/google-auth/index).
-
-#### MFA Using Duo Security
+## MFA Using Duo Security
 
 Duo Security allows you to request either of the following as your second factor once the user has provided their initial login credentials:
 
-* A user response to a push notification sent to the appropriate device;
-* A passcode provided to the user via SMS.
+* A user response to a push notification sent to the appropriate device
+* A passcode provided to the user via SMS
 
-You will find further instructions on enabling this feature [here](/multifactor-authentication/duo-security/index).
+[Click here to learn more about enabling Duo](/multifactor-authentication/duo)
 
-### MFA Using Custom Rules
+## MFA Using Custom Rules
 
-You may configure [rules](/rules) for custom MFA processes, which allow you to define the conditions that will trigger additional authentication challenges, such as changes in geographic location or logins from unrecognized devices.
+You may [configure rules](/rules) for custom MFA processes, which allow you to define the conditions that will trigger additional authentication challenges, such as changes in geographic location or logins from unrecognized devices.
 
-You will find [sample code snippets](/multifactor-authentication/custom/index) to assist you in building your rules here.
+[Click here for sample code snippets](/multifactor-authentication/custom) to assist you in building your rules here.
 
-### MFA Using Custom Providers
+## MFA Using a Custom Provider
 
 For a detailed look at implementing MFA with **YubiKey**, see [Multifactor Authentication with YubiKey-NEO](/multifactor-authentication/yubikey).
```

Comment:
For a detailed look at implementing a custom MFA provider, see [Multifactor Authentication with YubiKey-NEO](/multifactor-authentication/yubikey). This could be useful to start with (or soemething like that)